<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>745</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>任务安排</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>731</width>
      <height>61</height>
     </rect>
    </property>
    <widget class="QPushButton" name="instruction">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>0</y>
       <width>191</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>使用说明</string>
     </property>
    </widget>
    <widget class="QToolButton" name="Main_set">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>-2</y>
       <width>51</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>设置</string>
     </property>
    </widget>
    <widget class="QPushButton" name="cal">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>0</y>
       <width>261</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>生成</string>
     </property>
    </widget>
    <widget class="QPushButton" name="new_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>231</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>添加新日程</string>
     </property>
    </widget>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>731</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>580</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>上一天</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>580</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>下一天</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>731</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>731</width>
      <height>61</height>
     </rect>
    </property>
    <widget class="Line" name="line_3">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>0</y>
       <width>16</width>
       <height>61</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="line_4">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>0</y>
       <width>16</width>
       <height>61</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>10</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;我的一天&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="task_name_displayer">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>0</y>
       <width>241</width>
       <height>54</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;任务&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>10</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;待安排任务&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>227</x>
      <y>190</y>
      <width>20</width>
      <height>451</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_6">
    <property name="geometry">
     <rect>
      <x>488</x>
      <y>190</y>
      <width>20</width>
      <height>431</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="Day_Name">
    <property name="geometry">
     <rect>
      <x>335</x>
      <y>190</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:700;&quot;&gt;周一&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="Theblock">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>240</y>
      <width>251</width>
      <height>151</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="Theblock_contents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>249</width>
       <height>149</height>
      </rect>
     </property>
     <widget class="QWidget" name="widget_3" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>251</width>
        <height>151</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QScrollArea" name="day_task">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>430</y>
      <width>251</width>
      <height>151</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="Thetask_contents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>249</width>
       <height>149</height>
      </rect>
     </property>
     <widget class="QWidget" name="widget_4" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>251</width>
        <height>151</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>400</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>时段任务</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>200</y>
      <width>231</width>
      <height>411</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Fixed_time">
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>30</y>
        <width>40</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>备注</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="task_note">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>90</y>
        <width>40</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QWidget" name="task_details" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>221</width>
        <height>321</height>
       </rect>
      </property>
      <widget class="QLabel" name="ddl_hard">
       <property name="geometry">
        <rect>
         <x>135</x>
         <y>79</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="ddl_needtime">
       <property name="geometry">
        <rect>
         <x>135</x>
         <y>224</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="ddl_sametime">
       <property name="geometry">
        <rect>
         <x>135</x>
         <y>6</y>
         <width>71</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>6</x>
         <y>224</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>所需时间</string>
       </property>
      </widget>
      <widget class="QLabel" name="ddl_ddltime">
       <property name="geometry">
        <rect>
         <x>135</x>
         <y>152</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>6</x>
         <y>152</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>ddl时间</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>6</x>
         <y>79</y>
         <width>61</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>精力消耗</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>6</x>
         <y>6</y>
         <width>81</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>支持同时做？</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Ddl_change">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>280</y>
         <width>80</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>修改</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Ddl_delete">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>280</y>
         <width>80</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>删除</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>0</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:700; color:#aa55ff;&quot;&gt;ddl任务&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Fixed_ddl">
     <widget class="QStackedWidget" name="stackedWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>231</width>
        <height>451</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="Fixed_time_2">
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>30</y>
          <width>40</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>备注</string>
        </property>
       </widget>
       <widget class="QTextBrowser" name="task_note_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>221</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>90</y>
          <width>40</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QWidget" name="task_details_2" native="true">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>221</width>
          <height>321</height>
         </rect>
        </property>
        <widget class="QLabel" name="label_13">
         <property name="geometry">
          <rect>
           <x>6</x>
           <y>224</y>
           <width>91</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>所需时间</string>
         </property>
        </widget>
        <widget class="QLabel" name="fixed_needtime_2">
         <property name="geometry">
          <rect>
           <x>135</x>
           <y>224</y>
           <width>71</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="fixed_hard_2">
         <property name="geometry">
          <rect>
           <x>135</x>
           <y>79</y>
           <width>71</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="label_14">
         <property name="geometry">
          <rect>
           <x>6</x>
           <y>152</y>
           <width>81</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>时间段</string>
         </property>
        </widget>
        <widget class="QLabel" name="fixed_time_range">
         <property name="geometry">
          <rect>
           <x>135</x>
           <y>152</y>
           <width>81</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="label_15">
         <property name="geometry">
          <rect>
           <x>6</x>
           <y>6</y>
           <width>81</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>支持同时做？</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_11">
         <property name="geometry">
          <rect>
           <x>6</x>
           <y>79</y>
           <width>71</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>精力消耗</string>
         </property>
        </widget>
        <widget class="QLabel" name="fixed_sametime_2">
         <property name="geometry">
          <rect>
           <x>135</x>
           <y>6</y>
           <width>81</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QPushButton" name="Fixed_time_change">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>280</y>
           <width>81</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>修改</string>
         </property>
        </widget>
        <widget class="QPushButton" name="Fixed_time_delete">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>280</y>
           <width>91</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>删除</string>
         </property>
        </widget>
       </widget>
       <widget class="QLabel" name="label_16">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>-5</y>
          <width>71</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:700; color:#aa55ff;&quot;&gt;定时任务&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="Fixed_ddl_2"/>
      <widget class="QWidget" name="page_2">
       <widget class="QGraphicsView" name="graphicsView_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>221</width>
          <height>401</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page">
     <widget class="QGraphicsView" name="graphicsView">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>221</width>
        <height>401</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QScrollArea" name="to_be_added">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>211</width>
      <height>351</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>209</width>
       <height>349</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
